exampleSite/config.toml

```diff
@@ -854,6 +854,18 @@ ignoreErrors = ["error-missing-instagram-accesstoken"]
         label = ""
         lightTheme = "github-light"
         darkTheme = "github-dark"
+      [params.page.comment.waline]
+        enable = false
+        #js = "https://cdn.jsdelivr.net/npm/@waline/client@latest"
+        js = "https://cdn.jsdelivr.net/npm/@waline/client/dist/Waline.min.js"
+		serverURL = "https://xxx.vercel.app"
+        visitor = true
+		dark = 'auto'
+      [params.page.comment.twikoo]
+        enable = false
+		js = "https://cdn.jsdelivr.net/npm/twikoo@1.4.1/dist/twikoo.all.min.js"
```

Comment:
I think the properties should be indented at the same level, they're all referenced to the `params.page.comment.twikoo` group.

Reply:
Done.